CHANGELOG.md

```diff
@@ -1,17 +1,19 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+and this project adheres to
+[Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
```

Comment:
Would be nice to have a before-after conversion similar to what I had in v0.3. Since there was just 1 method before this PR this should be easy. I have a code example [here](https://github.com/AntoniosBarotsis/resend-rs/blob/88f3d18b3c471b87f5b8d3281c948e82ae21a9d0/src/resend_client.rs#L13-L26).

Reply:
The weird formatting is a result of me using deno fmt for markdown files.
I'm a bit confused, do you want an example in the client documentation?

Reply:
I think Github likes showing 6 lines minimum because I specifically selected the "unreleased" line. Meant to add an example similar to this: 
![image](https://github.com/AntoniosBarotsis/resend-rs/assets/50240570/ad947841-c758-45c1-8fd2-afb8d54d7e67)